test/langtools/tools/javac/doctree/DocCommentTester.java

```diff
@@ -727,7 +727,9 @@ public Void visitOther(DocTree node, Void p) {
              */
             void header(DocTree node) {
                 indent();
-                out.println(simpleClassName(node) + "[" + node.getKind() + ", pos:" + ((DCTree) node).pos);
+                var n = (DCTree) node;
+                out.println(simpleClassName(node) + "[" + node.getKind() + ", pos:" + n.pos +
+                        (n.getPreferredPosition() > n.pos ? ", prefPos:" + n.getPreferredPosition() : ""));
```

Comment:
Being paranoid, I suggest using `!=` instead of `>`

Reply:
If this suggestion of yours passes tests, I will integrate it.